app/models/book.py

```diff
@@ -12,4 +12,15 @@ def to_dict(self):
         book_as_dict["title"] = self.title
         book_as_dict["description"] = self.description
 
-        return book_as_dict
\ No newline at end of file
+        return book_as_dict
+    
+    @classmethod
+    def from_dict(cls, book_data):
+        new_book = Book(title=book_data["title"],
+                        description=book_data["description"])
+        # We could also use the `cls` keyword in place of `Book`
+        # The following declaration is equivalent to the one above
+        # new_book = cls(title=book_data["title"],
+        #               description=book_data["description"])
```

Comment:
Personally, I prefer using the `cls` approach rather than the literal name. While not important in this example, when using class methods to create what are effectively alternative constructors, using `cls` is more robust to inheritance. If we don't use `cls` that invites questions about why include a parameter we're not using (e.g. `@staticmethod` doesn't pass the class as the first param).

So I'd prefer showing the `cls` version as the "real" way, and  have the comment explain that the class type is passed in, so it's as though we had written `Book`.